### ISSUE TITLE
how to use 0.3 as a embedded database

### ISSUE THREAD

**Issue by gcxfd** [engula/engula#535]
as title

**Comment by huachaohuang** [comment]
I don't think Engula can be used as an embedded database in the short term. We are focusing on providing a database service now.

**Comment by gcxfd** [comment]
i see https://www.engula.io/blog/tutorial-0.2 can use as  a embedded database

maybe just export api without grpc ?  

I think there are a number of scenarios where this could be used

**Comment by huachaohuang** [comment]
Yes, I think an embedded database is very useful too. The design and implementation of Engula still make it possible to be used as an embedded database. But we still need some time to make that happen and the current direction focuses on the service part.

I think RocksDB and sled.rs are some awesome embedded databases. Have you considered using them?

**Comment by gcxfd** [comment]
I kown those , but I want something like https://vedis.symisc.net

**Comment by huachaohuang** [comment]
OK, so you want an embedded database with data structure APIs.
